packages/js/src/workouts/tailwind-components/base/radio-group.js

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable no-undefined */
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import { useCallback } from "@wordpress/element";
+
+const radioClassNameMap = {
+	variant: {
+		"default": "",
+		"inline-block": "yst-radio--inline-block",
+	},
+};
+
+
+const classNameMap = {
+	variant: {
+		"default": "",
+		"inline-block": "yst-radio-group--inline-block",
+	},
+};
+
+/**
+ * @param {string} id Identifier.
+ * @param {string} name Name.
+ * @param {string} value Value.
+ * @param {JSX.string} label Label.
+ * @param {JSX.string} srLabel Screen reader label.
+ * @param {string} [variant] Variant.
+ * @param {string} [className] CSS class.
+ * @returns {JSX.Element} Radio component.
+ */
+const Radio = ( {
+	id,
```

Comment:
We usually put all the parameters on the declaration line

Reply:
Not entirely true, I have seen this pattern a lot too. In fact, this is copied code 😉